fastparquet/util.py

```diff
@@ -106,3 +122,21 @@ def index_like(index):
                 index._start == 0 and
                 index._stop == len(index) and
                 index._step == 1 and index.name is None)
+
+
+def check_column_names(columns, *args):
+    """Ensure that parameters listing column names have corresponding columns"""
+    for arg in args:
+        if isinstance(arg, (tuple, list)):
```

Comment:
Things that are not tuple or list do automatically pass, by design. 
A few possible arguments can be bool or string as well as a list of columns.
(yes, this is my code)